src/frontend/components/Navbar/Navbar.jsx

```diff
@@ -18,58 +19,77 @@ const Navbar = () => {
       username: "",
     }));
   };
-  const [{ wishlist, cart }] = useProduct();
+  const [{ cart, wishlist }] = useProduct();
+  const [isMobileNavOpen, setIsMobileNavOpen] = useState(false);
   return (
-    <header className="navbar-wrapper d-flex align-center justify-around box-shadow-lg">
-      <Link
-        className="brand-name text-dec-none headline-lg font-wt-bold p-4"
-        to="/"
-      >
-        Gamers Zone
-      </Link>
-      <button id="hamburger-btn" className="btn text-md ml-auto mr-8 p-4">
-        <i className="fas fa-bars hamburger-icon"></i>
-      </button>
-      <section id="nav-menu-without-hamburger" className="d-flex m-2">
-        <nav className="nav-links d-flex align-center">
-          <NavItem
-            navItemTitle={"Products"}
-            icon={"store"}
-            navigateTo={"/products"}
-          />
-          <NavItem
-            number={cart.length}
-            navItemTitle={"Cart"}
-            icon={"shopping-cart"}
-            navigateTo={"/cart"}
-          />
-          <NavItem
-            number={wishlist.length}
-            navItemTitle={"Wishlist"}
-            icon={"heart"}
-            navigateTo={"/wishlist"}
-          />
-          <div className="ml-4">
-            {status === true ? (
-              <button
-                className="btn text-dec-none btn-primary rounded-sm text-sm p-4 mr-4"
-                onClick={() => signOutHandler(setAuth)}
-              >
-                Signout
-              </button>
-            ) : (
-              <Link
-                to="/signin"
-                className="btn text-dec-none btn-primary rounded-sm text-sm p-4 mr-4"
-              >
-                Signin
-              </Link>
+    <>
+      <header className="navbar-wrapper d-flex align-center justify-around box-shadow-lg">
+        <Link
+          className="brand-name text-dec-none headline-lg font-wt-bold p-4"
+          to="/"
+        >
+          Gamers Zone
+        </Link>
+        <button
+          id="hamburger-btn"
+          className="btn text-md ml-auto mr-8 p-4"
+          onClick={() => setIsMobileNavOpen(!isMobileNavOpen)}
+        >
+          <i className="fas fa-bars hamburger-icon"></i>
+        </button>
+        <section id="nav-menu-without-hamburger" className="d-flex m-2">
+          <nav className="nav-links d-flex align-center">
```

Comment:
I have observed you have used same nav code in HamburgerNav.jsx. If possible create another component and reuse it.

Reply:
will think about it